PRACE workshop blog post (#170)
MDAnalysis/MDAnalysis.github.io: _posts/2021-04-09-prace-workshop.md

---
layout: post
title: MDAnalysis Workshop @ PRACE / SURF
---

MDAnalysis will be delivering a 3 half-day remote workshop hosted by PRACE / SURF on the 26-28th of May.

> Workshop topics will range from the fundamentals of MDAnalysis all the way through to how to create your own AnalysisBase-derived methods and parallel analysis in a HPC setting.

Reviewer: ```suggestion
The workshop will include both lecture and practical portions. We will cover topics including:

* The fundamentals and basics of MDAnalysis
* How to use existing analyses
* How to write your own analyses
* Parallel analysis in an HPC setting
* Advanced functionality such as universe construction and transformations
```

Author: Please just push directly to the commit :P

## Register soon [places limited]

> The workshop is open to everyone, but places are limited, apply early!

Reviewer: Our target audience is users in academia and industry who are already familiar with Python and molecular dynamics. 

^ Can we shove that in anywhere?

---
layout: post
title: MDAnalysis Workshop @ PRACE / SURF
---

MDAnalysis will be delivering a 3 half-day remote workshop hosted by PRACE / SURF on the 26-28th of May.

The workshop will include both lecture and practical portions. We will cover topics including:

* The fundamentals and basics of MDAnalysis
* How to use existing analyses
* How to write your own analyses
* Parallel analysis in an HPC setting
* Advanced functionality such as universe construction and transformations

> 

Reviewer: ```suggestion
The workshop is aimed at users in academia and industry who are already familiar with Python and molecular dynamics. As many people may have different levels of experience with MDAnalysis, each session is separate.
```

## Register soon [places limited]

> The workshop is open to everyone, but places are limited, apply early!

Reviewer: ```suggestion
Places are limited. Please register early! You can sign up for individual sessions if you are only interested in certain topics.
```

Author: Is that true? I thought you had to sign up to the whole thing.

Reviewer: Oh. Err I thought it was. Let's leave it as a suggestion for now

Author: I can't see anything on the registration form that allows you to do that, thoughts @mnmelo ?

Reviewer 2: I don't think we can do separate registrations.

---
layout: post
title: MDAnalysis Workshop @ PRACE / SURF
---

MDAnalysis will be delivering a 3 half-day remote (online) workshop hosted by [PRACE](https://prace-ri.eu/) / [SURF](https://www.surf.nl/en) on the 26-28th of May 2021.

The workshop will include both lecture and practical portions. We will [cover topics](https://events.prace-ri.eu/event/1201/timetable/) including:

* The fundamentals and basics of MDAnalysis
* How to use existing analyses
* How to write your own analyses
* Parallel analysis in an HPC setting
* Advanced functionality such as universe construction and transformations

The workshop is aimed at users in academia and industry who are already familiar with Python and molecular dynamics. 

> As many people may have different levels of experience with MDAnalysis, we will have independent sessions ranging from introductory to advanced use.

Reviewer 2: Rewrote @lilyminium 's original sentence to avoid "separate sessions" because we do not have individual per-session registration, only full workshop.

Reviewer: I'm not sure what you are trying to say here -- if people sign up for all the sessions at once, isn't this functionally a crash course aimed at bringing introductory users up to advanced?

> MDAnalysis developers will be present to help you during practical portions of the sessions and to answer your questions about how to work productively with MDAnalysis.

Reviewer 3: I think it makes a good selling point to mention we'll be holding office hours for extended tutoring, including help with the design of the participants' own workflows.

* The fundamentals and basics of MDAnalysis
* How to use existing analyses
* How to write your own analyses
* Parallel analysis in an HPC setting (using SURFsara supercomputing facilities)
* Advanced functionality such as universe construction and transformations

The workshop is aimed at users in academia and industry who are already familiar with Python and molecular dynamics.
We will have separate sessions at different levels, ranging from introductory to
advanced use.
MDAnalysis developers will be present to help you during practical portions of the sessions and to answer your questions about how to work productively with MDAnalysis.
We will also be holding office hours for extended tutoring -- ask us for help with

> the design of the your own workflows!

Reviewer 3: 'the your' typo.

Reviewer: oops! fixed :-)
